### PR TITLE
[RUN-9510] XGBoost and submit-dist changes

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/admin/integration/deepspeed.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Use the *UI*, *CLI*, or the *API* to run the workload.
 Using the *CLI*:
 
 ```
-runai submit-mpi \--processes 2 -i
+runai submit-dis mpi \--processes 2 -i
```
Comment (itayvallach) [inline docs/admin/integration/deepspeed.md:88]: dist

Comment (itayvallach) [reply docs/admin/integration/deepspeed.md:88]: --workers

File: docs/home/whats-new-2-13.md [hunk 2]
```diff
@@ -88,6 +88,9 @@ The association between workspaces and node pools is done using *Compute resourc
 
 * Added integration with Weights & Biases Sweep to allow data scientists to submit hyperparameter optimization workloads directly from the Run:ai UI. To configure sweep, see [Sweep configuration](../admin/integration/weights-and-biases.md#sweep-configuration).
 
+<!-- RUN-9510 -->
+* Added support for NVIDIA XGBoost. XGBoost, which stands for Extreme Gradient Boosting, is a scalable, distributed gradient-boosted decision tree (GBDT) machine learning library. It provides parallel tree boosting and is the leading machine learning library for regression, classification, and ranking problems. For more information, see [runai submit-dist xgboost](../Researcher/cli-reference/runai-submit-dist-xgboost.md)
```
Comment (itayvallach) [inline docs/home/whats-new-2-13.md:92]: remove the word NVIDIA

File: docs/home/whats-new-2-10.md [hunk 3]
```diff
@@ -150,11 +150,11 @@ Added inactivity timeout for automated logout. The inactivity timeout is configu
 
 **Pytorch**
 
-Added CLI support for submitting Pytorch jobs. For more information, see [Submit Run:ai Pytorch job](../Researcher/cli-reference/runai-submit-pytorch.md).
+Added CLI support for submitting Pytorch jobs. For more information, see [Submit Run:ai Pytorch job](../Researcher/cli-reference/runai-submit-dist-pytorch.md).
```
Comment (itayvallach) [inline docs/home/whats-new-2-10.md:153]: Pytorch -> PyTorch

File: docs/admin/integration/deepspeed.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/admin/integration/deepspeed.md:88]:
```suggestion
runai submit-dist mpi \--workers 2 -i
```

File: docs/home/whats-new-2-13.md (hunk 2 above)
Comment (jasonnovichRunAI) [reply docs/home/whats-new-2-13.md:92]:
```suggestion
* Added support for XGBoost. XGBoost, which stands for Extreme Gradient Boosting, is a scalable, distributed gradient-boosted decision tree (GBDT) machine learning library. It provides parallel tree boosting and is the leading machine learning library for regression, classification, and ranking problems. For more information, see [runai submit-dist xgboost](../Researcher/cli-reference/runai-submit-dist-xgboost.md)
```

**Comment by omerbenedictrunai** [comment]
no comment

File: docs/Researcher/cli-reference/runai-submit-dist-xgboost.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+
+> The CleanPodPolicy controls deletion of pods when a job terminates. The policy can be one of the following values:
+>
+>* **Running**&mdash;only pods still running when a job completes (for example, parameter servers) will be deleted immediately. Completed pods will not be deleted so that the logs will be preserved. (Default)
+>* **All**&mdash;all (including completed) pods will be deleted immediately when the job finishes.
+>* **None**&mdash;no pods will be deleted when the job completes.
+
+#### --non-preemptible
+
+> Resources for non-preemptible jobs are guaranteed and will not be reclaimed at any time
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit-dist-xgboost.md:31]: @itayvallach we agreed to take this out, in the code & the documentation @jasonnovichRunAI

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ runai submit-tf --name distributed-job --replicas=2 -g 1 \
 
 > Resources for non-preemptible jobs are guaranteed and will not be reclaimed at any time
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit-dist-TF.md:36]: same here @itayvallach @jasonnovichRunAI

File: docs/Researcher/cli-reference/runai-submit-dist-mpi.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ You can start an unattended mpi training Job of name dist1, based on Project *te
 
 > Resources for non-preemptible jobs are guaranteed and will not be reclaimed at any time.
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit-dist-mpi.md:35]: Same here @itayvallach @jasonnovichRunAI

File: docs/Researcher/cli-reference/runai-submit-dist-pytorch.md [hunk 7]
```diff
@@ -42,7 +42,7 @@ runai submit-pytorch --name distributed-job --replicas=2 -g 1 \
 
 > Resources for non-preemptible jobs are guaranteed and will not be reclaimed at any time
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit-dist-pytorch.md:43]: Same here @itayvallach @jasonnovichRunAI

Comment (itayvallach) [reply docs/Researcher/cli-reference/runai-submit-dist-pytorch.md:43]: Correct, @jasonnovichRunAI  let's remove all occurrences of `--non-preemptible`

**Comment by itayvallach** [comment]
@jasonnovichRunAI  why haven't this been merged yet?

**Comment by jasonnovichRunAI** [comment]
> @jasonnovichRunAI why haven't this been merged yet?

Was waiting on PM approval. But if you are ok with all the changes, then I will merge.
